Lib/http/server.py

```diff
@@ -80,13 +61,12 @@
 # (Actually, the latter is only true if you know the server configuration
 # at the time the request was made!)
 
-__version__ = "0.6"
+__version__ = "0.7"
```

Comment:
Let's not bother bumping this, the important version is the Python version.

```suggestion
__version__ = "0.6"
```